bundle/manifests/manager-role_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Should we use the existing clusterroles/bindings in the config/rbac instead of adding new ones? so we don't have to maintain roles in multiple places.